packages/blocks/src/components/format-quick-bar/index.ts

```diff
@@ -30,6 +32,10 @@ export const showFormatQuickBar = async ({
   container?: HTMLElement;
   abortController?: AbortController;
 }) => {
+  // Abort previous format quick bar
+  globalAbortController.abort();
+  globalAbortController = abortController;
```

Comment:
maybe undefined?

Reply:
the `abortController` has default parameters, and there are TypeScript guards.
https://github.com/toeverything/blocksuite/blob/7a691c366fbfb923919ca592b59a813c73047cca/packages/blocks/src/components/format-quick-bar/index.ts#L23

Reply:
oh i see